scripts/linux/cross-platform-rust-build.sh

```diff
@@ -0,0 +1,295 @@
+#!/bin/bash
```

Comment:
I don't think we should consolidate this with the script in edgelet because there is a lot of edgelet specific logic.